scripts/prepare_navit_macos.sh

```diff
@@ -0,0 +1,19 @@
+# Use this script to build navit on a Mac
+/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+brew install glib gtk+ gettext libpng protobuf-c cmake librsvg imagemagick gpsd
+brew services start gpsd
+
+#TODO: this needs to be adapted for the PR: git clone https://github.com/navit/navit.git
+git clone https://github.com/OLFDB/navit.git
```

Comment:
don't checkout the code from here, prepare should be run from the git checkout already, so no need to download something again.